src/pages/Settings.vue

```diff
@@ -96,8 +96,9 @@ export default {
   },
   methods: {
     editDeviceName () {
-      this.editedDeviceName = this.settings.deviceName
-      this.isEditingDeviceName = true
+      // temporarily disabled until hpos bug is fixed
```

Comment:
Can you also skip the test related to this data (the `handles devicename interactions correctly` test) and write a note stating why it's skipped for now...?

Reply:
Right, good catch.